torchrl/envs/common.py

```diff
@@ -2393,14 +2393,16 @@ def rollout(
                 the call to ``rollout``.
 
         Keyword Args:
-            auto_reset (bool, optional): if ``True``, resets automatically the environment
-                if it is in a done state when the rollout is initiated.
-                Default is ``True``.
+            auto_reset (bool, optional): if ``True``, the contained environments will be reset before starting the
+                rollout. If ``False``, then the rollout will continue from a previous state, which requires the
+                ``tensordict`` argument to be passed with the previous rollout. Default is ``True``.
             auto_cast_to_device (bool, optional): if ``True``, the device of the tensordict is automatically cast to the
                 policy device before the policy is used. Default is ``False``.
-            break_when_any_done (bool): breaks if any of the done state is True. If False, a reset() is
-                called on the sub-envs that are done. Default is True.
-            break_when_all_done (bool): TODO
```

Comment:
Took the liberty of adding a docstring here and making some things a bit clearer. Open to suggestions for improvements!

Reply:
haha yeah much better! 
![200w](https://github.com/user-attachments/assets/c1236af1-9f49-4df2-b8eb-d8f26a3407ac)